rolocController.cpp

```diff
@@ -523,12 +538,41 @@ void ROLOCcontroller::onFreqSetTimerExpired()
     mpFreqencySetTimer->stop();
 }
 
+/**
+ * @brief ROLOCcontroller::rolocSetDepthMode - set depth mode
+ * @param mode - mode enum
+ */
+void ROLOCcontroller::rolocSetDepthMode(ROLOC::eLINEFINDER_MODE mode)
+{
+    Q_UNUSED(mode);
+    qint16 depthPacket = 0;
+    depthPacket = 0x0200;
+    depthPacket |= mFrequency;
+    int status = m_i2cBus.i2c_writeWord(mI2cAddr, ROLOC::eCMD_INFO, depthPacket);
+    if (status == BAD_DATA_READ)
+    {
+#if DBG_BLOCK
+    qDebug() << "write status --> " << status;
+#endif
+    }
+    mCurrentMode = ROLOC::eMODE_GET_DEPTH_MEASUREMENT;
```

Comment:
This if block will ALWAYS have the status = -2. So you dont need to DBG_BLOCK it. In fact, I would suggest making it a warning. 

```suggestion
    if (status == BAD_DATA_READ)
    {
#if DBG_BLOCK
    qWarning() << "write status i2c failed ";
#endif
    }
    mCurrentMode = ROLOC::eMODE_GET_DEPTH_MEASUREMENT;
```